package.json

```diff
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@stoplight/spectral-core": "^1.18.3",
+    "@stoplight/spectral-runtime": "^1.1.2",
```

Comment:
Why was this dependency added? I guess this has something to do with the exclusion of the pnpm lockfile from git. Whenever a dependency is changed the lockfile should be updated as well which isn't the case here. Please use npm when working within this project to not corrupt the existing package-lock.json.